Build fails

```
...
headings=[<h2 id="sublime.CompletionItem">
<code>sublime.CompletionItem</code> Class
            <span class="build" title="Added in build 4050">4050</span>
</h2>]
<td class="mth" data-skip="1">CompletionItem(trigger, &lt;annotation&gt;, &lt;completion&gt;, &lt;completion_format&gt;, &lt;kind&gt;)</td>
<td class="mth" data-qual-name="sublime.CompletionItem.CompletionItem.snippet_completion(trigger, snippet, &lt;annotation&gt;, &lt;kind&gt;)">CompletionItem.snippet_completion(trigger, snippet, &lt;annotation&gt;, &lt;kind&gt;)</td>
<td class="mth" data-qual-name="sublime.CompletionItem.CompletionItem.command_completion(trigger, command, &lt;args&gt;, &lt;annotation&gt;, &lt;kind&gt;)">CompletionItem.command_completion(trigger, command, &lt;args&gt;, &lt;annotation&gt;, &lt;kind&gt;)</td>

headings=[<h2 id="type-event_dict"><code>event</code> Dict</h2>]
Unexpected text: <h2 id="type-event_dict"><code>event</code> Dict</h2>
make: *** [fix-html] Error 1
```

--- a/prefix_methods.py
+++ b/prefix_methods.py
@@ -23,6 +23,7 @@ KEYWORD_MAP = {
     "Class": "class",
     "Value": "type",
     "Tuple": "type",
+    "Dict": "type",
 }
 
 

